lib/child_process.js

```diff
@@ -258,26 +251,20 @@ exports.execFile = function(file /*, args, options, callback*/) {
         if (!encoding)
           _stdout.push(chunk);
         else
-          _stdout += chunk;
+          _stdout += chunk.toString(encoding);
```

Comment:
Ah, probably didn't catch it because `'foo'.toString('whatever')` is still `'foo'`...